.github/workflows/multicore_darwin.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: build
-      run: ./build all-test multicore-darwin-x86_64 -g -j3 --with-production --enable-tracing
+      run: ./build LIBS multicore-darwin-x86_64 -g -j3 --with-production --enable-tracing
```

Comment:
Why is this changed to LIBS?

Reply:
Because we need to add "-tracemode projections" to the build command for the tests, which the build command doesn't do (and the make commands below wouldn't rebuild the tests since all-tests was done already).